cubic-server/world_storage/ChunkColumn.cpp

```diff
@@ -606,4 +609,77 @@ void ChunkColumn::processRandomTick(uint32_t rts)
     }
 }
 
+nbt_tag_t *ChunkColumn::toRegionCompatibleFormat()
+{
+    int32_t dataVersion = 2865; // TODO(huntears): This sucks, no idea what that value is
```

Comment:
![image](https://github.com/CubicMC/cubic-server/assets/60519602/e36e5350-052e-4734-9727-093c30f9f299)
I have this on a 1.19.3 save if you want

Reply:
tbh i don't think it really matters that much but I will update it.